Purchases/FoundationExtensions/String+Extensions.swift

```diff
@@ -16,10 +16,29 @@ import Foundation
 
 extension String {
 
+    enum StringError: Error {
```

Comment:
`String.StringError` seems redundant. You could do `enum Error: Swift.Error` so it becomes `String.Error`

Reply:
✅